### PR TITLE
DF-1664: Footer Update

### PR DESCRIPTION
**Pull request by sebworks**
Update Footer to match new design

## Changes

- Updates ' _includes/footer.html' file to match new design.
- Updates ' static/css/footer.less' file to match new design.

## Review

- @jimmynotjim 
- @anselmbradford 
- @ajbush 

## Preview
 
screen shot 2015-03-18 at 10 03 10 am

### CODE REVIEW

**Comment by anselmbradford** [comment]
LGTM :+1:

File: _includes/footer.html [hunk 1]
```diff
@@ -178,7 +162,7 @@ <h2 class="h6">
                         Washington, D.C. 20552
                     </p>
                     <div class="footer_general-inqueries">
-                        <b class="footer_general-inqueries-label">For general inquiries only</b>
+                        <b class="footer_general-inqueries-label"><i>For general inquiries only</i></b>
```
Comment (jimmynotjim) [inline _includes/footer.html]: Should we really be using `<b>` and `<i>`? It might be better to hook into the `.footer_general-inqueries-label` class.

Comment (Scotchester) [inline _includes/footer.html]: Looks to me like it should be a heading, too.

Comment (sebworks) [inline _includes/footer.html]: ok, I will make it a heading and move they styles to a class.

Comment (sebworks) [inline _includes/footer.html]: @himedlooff, would love your input here

Comment (himedlooff) [inline _includes/footer.html]: A heading would be more consistent. Most of the other footer areas use them.

**Comment by anselmbradford** [comment]
Is the address and general inquiries supposed to be side-by-side at desktop sizes? That's not how it is in the original screenshot.
screen shot 2015-03-19 at 9 47 04 am

_(Also—if it isn't too much trouble!—it would be easier if the commits were squashed just before merging so that just the code review edits can be checked in isolation. I think then the branch edits can be pulled down without running into a conflict also.)_

**Comment by Scotchester** [comment]
BTW, we don't load Avenir Next with italics in any weight other than normal, so it's faux-italicizing this here.

**Comment by anselmbradford** [comment]
Running `grunt` solved it.
screen shot 2015-03-19 at 10 02 39 am

:+1: unless the faux-italicizing is a problem.

**Comment by sebworks** [comment]
@Scotchester that is true but it's either .web-demi() or web-italic(). Which should I use?

**Comment by Scotchester** [comment]
Sorry I lost track of this question, but I'd say that it should be answered by your designers.

**Pull request merged**